public/app/features/alerting/unified/components/rule-editor/GrafanaEvaluationBehavior.tsx

```diff
@@ -52,6 +55,16 @@ const evaluateEveryValidationOptions: RegisterOptions = {
   },
 };
 
+// TODO Move to more generic place
```

Comment:
If we want to use this method as a generic method, maybe it's a good idea to inject `minInterval` as a parameter, so we have a pure function?